docs/index.md

```diff
@@ -176,11 +180,59 @@ So let's rename this goal statement to `2p2e4`:
 > to the left of the first statement).
 > Change the name of the statement from "1" to "2p2e4" and press Enter (Return).
 
+#### Interlude: Brief review of metamath-lamp user interface
+
+Let's look at the display we have so far
+(your screen may look somewhat different):
+
+![Metamath-lamp display with just the 2p2e4 goal](2p2e4_goal.png)
+
+The top line summarizes the context - we loaded the `set.mm` database
+(classical logic and ZFC set theory) and stopped reading the database before
+`2p2e4`.
+
+The next line is a tab bar, letting you select between *Settings* and
+*Editor*.
+If you want to change the configuration of the tool, use *Settings*.
+Normally you''l be in the *Editor* tab (which lets you edit proofs),
+so we'll focus on that.
+
+Below the word "Editor" is an icon bar that shows different icons;
+each icon represents an actions you can do in the editor.
+We've already used one, the "+" symbol that represents adding a statement.
+The reference manual section
+[Editor command tab bar](#editor-command-tab-bar) discusses the
+icons and actions in more detail.
+You can hover over an icon to see what the command does.
+
+Here are some key icons and their meanings:
+
+* Box: Select or deselect all current statements.
+* Up and Down: Move the selected statements up or down in their list.
+* "+": Add a statement (which you then type in).
+* Trash can: Delete the selected statement(s).
+* Duplicated "+": Copy the selected statement.
+* Merge: Merge the selected statements (they must be similar).
```

Comment:
```
Merge: Merge the selected statements (they must be similar).
```

It might be worth adding (please feel free to rephrase it, or skip adding it at all if you think such detalization is not required in this place):

```
Merge: Merge the selected statements (they must be similar). 
Before clicking this button select only one statement, 
the other similar one will be detected by metamath-lamp.
```

Reply:
Thanks! I think that added statement doesn't belong here, as it's just trying to hint at what they are. I instead added that statement to the reference manual section.